kolibri/plugins/user/assets/src/constants.js

```diff
@@ -6,4 +6,6 @@ const PageNames = {
   PROFILE: 'PROFILE',
 };
 
-export { PageNames };
+const signedOutDueToInactivity = 'SIGNED_OUT_DUE_TO_INACTIVITY';
```

Comment:
Shouldn't this be in core constants, because it will have to be set in the Lockr by a process external to the user plugin?